openupgrade_scripts/scripts/payment/16.0.2.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,417 @@
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / capture_manually (boolean)    : DEL
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / color (integer)               : DEL
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / company_id (many2one)         : DEL relation: res.company, required
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / country_ids (many2many)       : DEL relation: res.country
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / description (html)            : DEL
+# NOTHING TO DO: feature lost.
+
+payment      / payment.acquirer         / display_as (char)             : DEL
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / done_msg (html)               : DEL
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / fees_active (boolean)         : DEL
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / fees_dom_fixed (float)        : DEL
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / fees_dom_var (float)          : DEL
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / fees_int_fixed (float)        : DEL
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / fees_int_var (float)          : DEL
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / image_128 (binary)            : DEL attachment: True
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / inline_form_view_id (many2one): DEL relation: ir.ui.view
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / module_id (many2one)          : DEL relation: ir.module.module
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / module_state (selection)      : DEL selection_keys: function
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / name (char)                   : DEL required
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / payment_icon_ids (many2many)  : DEL relation: payment.icon
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / pending_msg (html)            : DEL
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / pre_msg (html)                : DEL
+# DONE: pre-migration: moved to payment.provider
+
+payment      / payment.acquirer         / provider (selection)          : DEL required, selection_keys: ['none']
+# NOTHING TO DO: Not needed anymore.
```

Comment:
This column still exists in the database after migration. Is that normal?

I do need this column to exist in my post-migration script in #3965, but it can be deleted afterwards as far as I'm concerned.

Reply:
> This column still exists in the database after migration. Is that normal?

Yes, OU doesn't remove any column by design for preserving the maximum amount of data. You can remove it afterwards passing `database_cleanup`.